src/core/server/integration_tests/saved_objects/routes/export.test.ts

```diff
@@ -15,7 +15,8 @@ import {
 } from '@kbn/core-usage-data-server-mocks';
 import { savedObjectsExporterMock } from '@kbn/core-saved-objects-import-export-server-mocks';
 import type { SavedObjectConfig } from '@kbn/core-saved-objects-base-server-internal';
-import { setupServer, createExportableType } from './test_utils';
+import { createExportableType } from './test_utils';
```

Comment:
I think this should also be available in the new package, right?

Reply:
I've left it together with saved_objects in core for now since we actually have two `test_utils` files: One in core/server and the other here, with the saved objects integration tests.

Reply:
It makes sense. However, I can see the method is actually migrated to the package https://github.com/elastic/kibana/pull/143483/files#diff-7466a8feb95b9d08b98bea557653c5211998d34f1d21bc74177b6c836b059d53R66-R78

The only thing is that it's not exported. Should we remove it from the package for now?

Reply:
Either way would work, it doesn't really matter while we're in the middle of the migration process. However, I thought about it some more and guess there's no harm in moving the method to a package now. It's generic enough and we'll need to figure it out anyway, so I'll follow through with your original request to move the method to the package. It also means we'll have a bit more freedom to move the integration test suites however we see fit in the future.